transform/mattermost-analytics/models/staging/stripe/stg_stripe__invoice_line_items.sql

```diff
@@ -0,0 +1,38 @@
+with source as (
+
+    select * from {{ source('stripe_raw', 'invoice_line_items') }}
+
+),
+
+invoice_line_items as (
+
+    select
+          amount as amount
+        , currency as currency
+        , description as description
+        , discountable as discountable
+        , discounts as discounts
+        , discount_amounts as discount_amounts
+        , id as id
+        , invoice as invoice
+        , invoice_item as invoice_item
+        , livemode as livemode
+        , metadata as metadata
```

Comment:
Let's extract the items needed from `metadata` here and avoid passing the column downstream. See the comment [here](https://github.com/mattermost/mattermost-data-warehouse/pull/1255#discussion_r1173396780) for more context.

This applies to other JSON fields as well.

Reply:
Removed this.